home-work-app/src/app/configs/style.config.scss

```diff
@@ -0,0 +1,36 @@
+// colors
+$color-black: #232323;
+$color-gray-tint1: rgb(66, 66, 66);
+$color-gray-tint2: #555555;
+$color-gray-tint2: #969696;
+$color-pink: rgb(246, 82, 97);
+$color-white: #ffffff;
+
+// default styles
```

Comment:
global styles could be applied with `createGlobalStyle` from `styled-components`

Reply:
Done